tests/ui/macros/builtin-env-issue-114010.stderr

```diff
@@ -0,0 +1,11 @@
+error: environment variable `oopsie` not defined at compile time
+  --> $DIR/builtin-env-issue-114010.rs:3:1
+   |
+LL | env![r#"oopsie"#];
+   | ^^^^^^^^^^^^^^^^^
+   |
+   = help: use `std::env::var("oopsie")` to read the variable at run time
```

Comment:
Should we print this as a raw string as well? Can you add a test that actually depends on the rendered str being raw?

Reply:
Fixed!